mint.json

```diff
@@ -53,7 +53,8 @@
         "introduction",
         "concepts",
         "installation",
-        "integration"
+        "integration",
+        "upgrades"
```

Comment:
could we change this to `upgrading`? I also wonder if it should go under the 'Operating' section?